src/less/icon/ds6/icon.less

```diff
@@ -1,6 +1,21 @@
 @import "../../less/mixins-common.less";
 
 svg.icon {
-    .svg-icon-default;
+    .svg-icon-base;
     vertical-align: middle;
 }
+
+svg.icon--chevron-dropdown {
+    height: 5px;
+    width: 8px;
+}
+
+svg.icon--arrow-cta {
+    height: 10px;
+    width: 9.75px;
+}
+
+svg.icon--hamburger {
+    height: 14px;
+    width: 12px;
+}
```

Comment:
So, these are the only three icons at the moment?

Reply:
Yeah, that's it. Waiting on design system to publish iconography.